CHANGELOG.md

```diff
@@ -152,12 +152,13 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (crypto) [#15070](https://github.com/cosmos/cosmos-sdk/pull/15070) `GenerateFromPassword` and `Cost` from `bcrypt.go` now take a `uint32` instead of a `int` type.  
 * (x/capability) [#15344](https://github.com/cosmos/cosmos-sdk/pull/15344) Capability module was removed and is now housed in [IBC-GO](https://github.com/cosmos/ibc-go). 
 * [#15299](https://github.com/cosmos/cosmos-sdk/pull/15299) remove `StdTx` transaction and signing APIs. No SDK version has actually supported `StdTx` since before Stargate.
-* [#15600](https://github.com/cosmos/cosmos-sdk/pull/15600) add support for getting signers to `codec.Codec` and protoregistry support to `InterfaceRegistry`:
+* (codec) [#15600](https://github.com/cosmos/cosmos-sdk/pull/15600) add support for getting signers to `codec.Codec` and protoregistry support to `InterfaceRegistry`:
     * `Codec` is now a private interface and has the methods `InterfaceRegistry`, `GetMsgAnySigners`, `GetMsgV1Signers`, and `GetMsgV2Signers` which will fail when using `AminoCodec`.
      All implementations of `Codec` by other users must now embed an official implementation from the `codec` package.
     * `InterfaceRegistry` is now a private interface and implements `protodesc.Resolver` plus the `RangeFiles` method
      All implementations of `InterfaceRegistry` by other users must now embed the official implementation.
     * `AminoCodec` is marked as deprecated.
+* (genesis) [#15826](https://github.com/cosmos/cosmos-sdk/pull/15826) Remove get address command from genesis cmd
```

Comment:
```suggestion
```

This should be removed, as this is not an API breaking change, neither a genesis change.
This should be in a ### CLI Breaking Changes section.
Moreover, the command removed is `simd q account`, which isn't related to genesis.
So the message should be something like "`simd q account` has been removed, please use `simd q auth account` instead"

Reply:
Removed the automerge label because of this comment is still applicable.